classes/mender-install.bbclass

```diff
@@ -13,11 +6,4 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 VIRTUAL-RUNTIME_initscripts = ""
 
-
-#Make sure we are creating sdimg with all needed partitioning.
-IMAGE_CLASSES += "sdimg"
-IMAGE_FSTYPES += "sdimg"
-
-IMAGE_INSTALL_append = " u-boot-fw-utils mender ca-certificates"
-
-
+IMAGE_INSTALL_append = " mender ca-certificates"
```

Comment:
This has bothered me for some time already. I'm not convinced that classes should be used to install packages into the image. In my opinion mender should be installed by `*.bbappend` to one of the base images.